web/src/main/webapp/resources/js/atsy-new-application-popup.js

```diff
@@ -0,0 +1,46 @@
+var positionDefaultValueInEng = 'Please choose a position';
+var applicationSourceDefaultValueInEng = 'Please choose an application source';
+var positionDefaultValueInHun = 'Kérem válasszon egy pozíciót';
+var applicationSourceDefaultValueInHun = 'Kérem válasszon egy jelentkezési forrást';
+
+function validateApplicationSourceAndPositionDropDowns() {
+    var positionIsValid = positionValidator();
+    var applicationSourceIsValid = applicationSourceValidator();
+    return (positionIsValid && applicationSourceIsValid);
+}
+
+function positionValidator() {
+    var ddl = document.getElementById('position');
+    var selectedValue = ddl.options[ddl.selectedIndex].value;
+
+    if (selectedValue == positionDefaultValueInEng) {
```

Comment:
Since the default value is the first option in the ddl you can simply check `ddl.selectedIndex === 0` and append `selectedValue` to `position_error`. Thus constants in line 1-4 are not needed.